Packs/Intezer/Integrations/IntezerV2/IntezerV2_test.py

```diff
@@ -229,7 +231,8 @@ def test_check_analysis_status_and_get_results_url_command_single_success(reques
                 'analysis_id': analysis_id,
                 'summary': {
                     'title': 'malicious',
-                    'verdict_name': 'malicious'
+                    'verdict_name': 'malicious',
+                    'verdict_type': 'malicious'
```

Comment:
I think it would be great to add to this test (or to another test) verification of the dbot score that was outputted to the context. I mean, I don't think that this test checks the dbot score that was generated (it checks only the dbot indicator).

Reply:
Done